docker-compose.yml

```diff
@@ -1,10 +1,11 @@
-version: '3'
+version: '3.5'
 
 services:
   jira:
     depends_on:
       - postgresql
     image: blacklabelops/jira
+    container_name: jira
     networks:
```

Comment:
I am against using container name in the docker-compose example.

I want to provide a docker-compose yaml that can be used to deploy multiple instances of Jira, not just one.